src/commands/serve.ts

```diff
@@ -48,16 +51,9 @@ export const builder = (yargs: Argv) =>
         describe: 'The relative path for storing ad4m data', 
         alias: 'rp'
       },
-      defaultLangPath: {
-        type: 'string',
-        describe: 'Path of the default languages used to start ad4m service',
-        default: '../../temp/languages',
-        alias: 'dlp'
-      },
       networkBootstrapSeed: {
```

Comment:
Is it better to put the logic in `init` command or maybe a new `bootstrap` command?

Reply:
I had put it in serve so that you have the ability to change the bootstrap seed that is used between sessions running ad4m-host. Useful for the case where you might want to switch between dev & prod bootstrap seeds, or perhaps update to a new bootstrap seed which fixes a bug in one language but does not have any other breaking changes.

Reply:
Ah never mind, I thought the ad4m-host init functions was the same as the executor init function. I can see that it just sets up dependencies. I agree better this lives in init.